app/uk/gov/hmrc/plasticpackagingtax/registration/config/ErrorHandler.scala

```diff
@@ -30,6 +30,21 @@ class ErrorHandler @Inject() (error_template: error_template, val messagesApi: M
   override def standardErrorTemplate(pageTitle: String, heading: String, message: String)(implicit
     request: Request[_]
   ): Html =
-    error_template(pageTitle, heading, message)
+    error_template(pageTitle, heading, List(message))
+
+  override def badRequestTemplate(implicit request: Request[_]): Html          = pptErrorTemplate()
+  override def notFoundTemplate(implicit request: Request[_]): Html            = pptErrorTemplate()
+  override def internalServerErrorTemplate(implicit request: Request[_]): Html = pptErrorTemplate()
```

Comment:
What are these 3 new over-rides for?

Reply:
To ensure we display the same content (as that displayed on the new error_page) on our generic error page.

Reply:
Was that part of the requirements?

Reply:
Not according to the Jira ticket but that's what I though we'd agreed verbally during our 3As call yesterday. Probably worth checking with Jon and Steve G.

Reply:
Ticket has been updated to reflect the want to have the generic error page updating.